integration-tests/src/test/java/org/apache/fineract/integrationtests/SavingsAccountTransactionTest.java

```diff
@@ -81,17 +101,86 @@ public void verifySavingsTransactionSubmittedOnDateAndTransactionDate() throws J
             LocalDate withdrawDate = Utils.getDateAsLocalDate(withdrawDateString);
 
             final Integer clientID = ClientHelper.createClient(this.requestSpec, this.responseSpec, startDateString);
-            Assertions.assertNotNull(clientID);
+            assertNotNull(clientID);
 
             final Integer savingsId = createSavingsAccountDailyPosting(clientID, startDateString);
-            Assertions.assertNotNull(savingsId);
+            assertNotNull(savingsId);
 
             performSavingsTransaction(savingsId, "100", depositDate, true);
             performSavingsTransaction(savingsId, "50", withdrawDate, false);
         } finally {
             enableBusinessDate(requestSpec, responseSpec, false);
         }
+    }
+
+    @Test
+    public void testConcurrentSavingsTransactions() {
+        final Integer clientID = ClientHelper.createClient(this.requestSpec, this.responseSpec);
```

Comment:
Can you please help me understand this test a little bit more? I was wondering what does it test exactly.